spec/features/bulk_updates/bulk_updates_data_centers_spec.rb

```diff
@@ -37,9 +35,12 @@
     it 'displays the preview information', bulk_update_step_1: true do
       expect(page).to have_content('Preview of New MMT_2 Bulk Update')
 
-      expect(page).to have_content("Name #{bulk_update_name}")
-      expect(page).to have_content('Field to Update Data Centers')
-      expect(page).to have_content('Update Type Find And Remove')
+      expect(page).to have_content('Name')
```

Comment:
talked on slack, but putting it here, we should explore, shortly, if this is the best way.

Reply:
I can't post the screenshot here, but if you look at the page, each of these text pieces are actually on separate lines, so I thought it made sense to actually separate them in the test. (even though they were on separate lines, Poltergeist treated the whitespace differently so it accepted the text on one line)

Reply:
I can see that, and I can see how different algorithms would treat 'space newline space' different then 'space space space' when dealing with HTML, with the old way we knew that "Field to Update" was followed by "Instruments" with only white space between them, where as here we are assuming that. But if normalize white space did not resolve this, then we should just keep it in mind that it could be faked or easily confused and move on.

So my question is, did the normalize white space parameter that I saw used in other cases not work here?